ethcore/res/ethereum/tests-issues/currents.json

```diff
@@ -0,0 +1,215 @@
+                                        "subnumbers": ["1"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "RevertDepth2": {
+                                        "subnumbers": ["1"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "RevertRemoteSubCallStorageOOG2": {
+                                        "subnumbers": ["1","2"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "RevertDepthCreateAddressCollision": {
+                                        "subnumbers": ["3","4"],
+                                        "chain": "Constantinople (test)"
+                                }
+                        }
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stStaticCall",
+                        "subtests": {
+                                "static_callCreate": {
+                                        "subnumbers": ["2"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "static_CheckOpcodes4": {
+                                        "subnumbers": ["3"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "static_CheckOpcodes3": {
+                                        "subnumbers": ["5","6","7","8"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "static_RevertDepth2": {
+                                        "subnumbers": ["1"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "static_callBasic": {
+                                        "subnumbers": ["1","2"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "static_CheckOpcodes2": {
+                                        "subnumbers": ["5","6","7","8"],
+                                        "chain": "Constantinople (test)"
+                                }
+                        }
+                },
+                {
+                        "reference": 9568,
+                        "failing": "stZeroKnowledge",
+                        "subtests": {
+                                "pointAddTrunc": {
+                                        "subnumbers": ["*"],
+                                        "chain": "Constantinople (test)"
+                                },
+                                "pointAdd": {
```

Comment:
This one is indeed `ethereum/tests`'s issue: https://github.com/ethereum/tests/issues/512